draft-kuhn-quic-bdpframe-extension-02.xml

```diff
@@ -368,7 +378,7 @@ the CC parameters -->
                 This section describes the use of a new Frame, the BDP_FRAME.
                 The BDP_FRAME can be utilized by the congestion controller
                 and its data is not be limited by flow control limits. The sender and the receiver MAY
-                send multiple BDP_FRAMEs in both 1-RTT and 0-RTT connections. 
+                send multiple BDP_FRAMEs in both 1-RTT and 0-RTT connections. The rate of update SHOULD be limited.
```

Comment:
SHOULD be limited ... to what ... could we say much less frequently than once for several RTTs?